doc/security/vulnerabilities.rst

```diff
@@ -1732,7 +1732,15 @@ This has been fixed in main for v3.7.0
 CVE-2024-4785
 -------------
 
-Under embargo until 2024-08-07
+Bluetooth: Missing Check in LL_CONNECTION_UPDATE_IND Packet Leads to Division by Zero
+
+- `Zephyr project bug tracker GHSA-xcr5-5g98-mchp
+  <https://github.com/zephyrproject-rtos/zephyr/security/advisories/GHSA-xcr5-5g98-mchp>`_
+
+This has been fixed in main for v3.7.0
```

Comment:
In main or in 3.7 too? phrasing is unclear.

Reply:
v3.7.0 is the tag, not the branch, i.e. it means "has been fixed in main and is available in v3.7.0 release"
But I tend to agree that the wording can be unclear, but it's basically the same form that's being used for all other disclosed vulnerabilities in this page.

Reply:
That is the pattern adopted. It means that it was fixed in the main branch during 3.7 cycle.